### ISSUE TITLE
"Input/output error" on container start

### ISSUE THREAD

**Issue by bedrich-schindler** [docker/for-mac#4998]
### Expected behavior

We have `docker-compose.yml` with following content:

```
web:
    image: visionappscz/apache-php:7.4
    ports:
        - 80:80
    volumes:
        - .:/www:z
```

After `docker-compose up` is executed, following outputs:

```
$docker-compose up --build

Starting ra2_ui_api2_web_1 ... done
Attaching to ra2_ui_api2_web_1
web_1  | usermod: no changes
web_1  | usermod: no changes
web_1  | chown: changing ownership of '/www/.git/objects/pack/pack-4d221ab71dd698637f2b6cd2403854233b12c497.idx': Input/output error
web_1  | chown: changing ownership of '/www/.git/objects/pack/pack-4d221ab71dd698637f2b6cd2403854233b12c497.pack': Input/output error
ra2_ui_api2_web_1 exited with code 1
```

This is happening on stable version 2.4.0.0 (48506), but it worked on previous versions (and it works on another systems as well). This is happening on multiple Macs.

### Actual behavior

Container starts as before version 2.4.0.0.

### Information

We use our image https://github.com/visionappscz/apache-php that runs init script after start: https://github.com/visionappscz/apache-php/blob/master/scripts/init-container.sh

It fails in this part. We use this kind of init script for years and it started to fail after update to version 2.4.0.0. And it works on Linux.

**Comment by stephen-turner** [comment]
Thanks for the report, @bedrich-schindler. I believe this is a dupe of #4989, so I'll close this copy. Please follow that ticket for updates.

**Comment by bedrich-schindler** [comment]
Thank you for your response. I wasn't sure if it is the same problem or not.

**Temporary fix until fixed in stable release is to downgrade to 2.3.0.5.**
